extensions/notification/utils/auth.js

```diff
@@ -64,6 +64,9 @@ function generateKafkaAuthObject(auth) {
         const keytabPath = getAuthFilePath(keytab);
         if (keytabPath) {
             authObject['sasl.kerberos.keytab'] = keytabPath;
+            // default kinit command
+            const kinitCommand = `kinit -k ${principal} -t ${keytabPath}`;
+            authObject['sasl.kerberos.kinit.cmd'] = kinitCommand;
```

Comment:
It's very important to address the TODO comment 
```
    // TODO: auth object is fluid at the moment, once a solid structure is
    // defined, introduce a schema
 ```
 , as I see there is no check to ensure principal or keytabPath are defined.

Reply:
Agreed, it was an unknown when we started development, now that we have a good understanding of the parameters involved, we should enforce schema or checks. I will create a new JIRA ticket/PR for this.